src/website/pages/index.tsx

```diff
@@ -35,6 +36,9 @@ export default function Home() {
           <DashboardContainer />
         </div>
       </div>
+      <div className={styles.rayecontainer}>
```

Comment:
As well as the `rayecontainer` -> `polarisContainer`

(Sorry) Could we also change `maincontainer` and `dashboardcontainer` to camel case as well?